collector/src/lib.rs

```diff
@@ -6,46 +6,56 @@
 //!
 //! # Examples
 //!
+//! Initialize an instance of [`Collector`].
+//!
 //! ```no_run
-//! use janus_collector::{AuthenticationToken, Collector};
-//! use janus_core::{hpke::generate_hpke_config_and_private_key};
-//! use janus_messages::{
-//!     Duration, HpkeAeadId, HpkeConfig, HpkeConfigId, HpkeKdfId, HpkeKemId, Interval, TaskId,
-//!     Time, Query,
-//! };
+//! use std::{fs::File, str::FromStr};
+//!
+//! use janus_collector::{Collector, PrivateCollectorCredential};
+//! use janus_messages::{Duration, FixedSizeQuery, Interval, Query, TaskId, Time};
 //! use prio::vdaf::prio3::Prio3;
-//! use rand::random;
 //! use url::Url;
 //!
 //! # async fn run() {
-//! // Supply DAP task parameters.
-//! let task_id = random();
-//! let hpke_keypair = janus_core::hpke::generate_hpke_config_and_private_key(
-//!     HpkeConfigId::from(0),
-//!     HpkeKemId::X25519HkdfSha256,
-//!     HpkeKdfId::HkdfSha256,
-//!     HpkeAeadId::Aes128Gcm,
-//! ).unwrap();
+//!     let task_id = TaskId::from_str("[your DAP task ID here]").unwrap();
```

Comment:
I'd recommend leaving this un-indented, since the function's opening and closing braces won't appear in the final output. (It's just present to allow us to use `.await` in this context)